src/bpm/commands/run.go

```diff
@@ -26,11 +26,18 @@ import (
 	"bpm/runc/lifecycle"
 )
 
-var volumes []string
+var (
+	// Volumes which come from command-line flags.
+	volumes []string
+
+	// Environment variables which come from command-line flags.
+	env []string
+)
 
 func init() {
 	runCommand.Flags().StringVarP(&procName, "process", "p", "", "the optional process name")
 	runCommand.Flags().StringSliceVarP(&volumes, "volume", "v", []string{}, "Optional list of volumes (format: <path>[:<options>])")
+	runCommand.Flags().StringSliceVarP(&env, "env", "e", []string{}, "Additional environment variables (format: KEY=VALUE")
```

Comment:
Is it okay to allow users to pass possible sensitive values as arguments? Alternatively strict option of, `SECRET=x bpm run stuff -e SECRET ...`.

Reply:
I think I first ran into it with `docker run` (they support both styles); and Concourse `fly execute` kind of does it. I'm not sure how much of a convention it is, but it's something I've thought useful and helps avoid exposing secrets through args.

Reply:
I've pulled this out into a [separate issue](https://github.com/cloudfoundry-incubator/bpm-release/issues/90). Since this would be a non-breaking change and the use-case driving this doesn't require secrets I'm leaning towards doing another release in the next few days which could/could not add this functionality.

I'll add a note to the docs for now.